articles/spring-cloud/quickstart-deploy-enterprise.md

```diff
@@ -0,0 +1,374 @@
+- A license for Azure Spring Cloud Enterprise Tier. For more information, see [View Azure Spring Cloud Enterprise Tier Offer in Azure Marketplace](./how-to-enterprise-marketplace-offer.md).
+- [The Azure CLI version 2.0.67 or higher](/cli/azure/install-azure-cli).
+- [Git](https://git-scm.com/).
+- [jq](https://stedolan.github.io/jq/download/)
+- [!INCLUDE [install-enterprise-extension](includes/install-enterprise-extension.md)]
+
+## Download the sample app
+
+Use a local terminal for the following steps.
+
+1. Create a new folder and clone the sample app repository.
+
+   ```bash
+   mkdir source-code
+   ```
+
+   ```bash
+   cd source-code
+   ```
+   ```bash
+   git clone https://github.com/Azure-Samples/acme-fitness-store
+   ```
+
+1. Navigate into the repository directory.
+
+   ```bash
+   cd acme-fitness-store
+   ```
+
+## Provision a service instance
+
+Use the following steps to provision an Azure Spring Cloud service instance:
+
+1. Update Azure CLI with the Azure Spring Cloud extension by using the following command:
+
+   ```azurecli
+   az extension update --name spring-cloud
+   ```
+
+1. Sign in to the Azure CLI and choose your active subscription by using the following command:
+
+   ```azurecli
+   az login
+   az account list --output table
+   az account set --subscription <subscription-ID>
+   ```
+
+1. Use the following command to accept the legal terms and privacy statements for the Enterprise tier. This step is necessary only if your subscription has never been used to create an Enterprise tier instance of Azure Spring Cloud.
+
+   ```azurecli
+   az provider register --namespace Microsoft.SaaS
+   az term accept --publisher vmware-inc --product azure-spring-cloud-vmware-tanzu-2 --plan tanzu-asc-ent-mtr
+   ```
+
+1. Select a location. This location must be a location supporting Azure Spring Cloud Enterprise tier.
+
+1. Create a resource group using the following command:
+
+    ```azurecli
+    az group create --name <resource-group-name> \
+        --location <location>
+    ```
+
+    For more information about resource groups, see [What is Azure Resource Manager?](../azure-resource-manager/management/overview.md).
+
+1. Prepare a name for your Azure Spring Cloud service instance. The name must be between 4 and 32 characters long and can contain only lowercase letters, numbers, and hyphens. The first character of the service name must be a letter and the last character must be either a letter or a number.
+
+1. Create an Azure Spring Cloud service instance using the following the command:
+
+   ```azurecli
+   az spring-cloud create \
+       --resource-group <resource-group-name> \
+       --name <service-instance-name> \
+       --sku enterprise \
+       --enable-application-configuration-service \
+       --enable-service-registry \
+       --enable-gateway \
+       --enable-api-portal
+   ```
+
+1. Set your default resource group name, location, and Spring Cloud service name using the following command:
```

Comment:
Don't use configured defaults. Instead, include the necessary parameters in every snippet block. The reader can decide to use defaults if they are aware and they prefer that, but for documentation, we prefer that everything is explicit.